### ISSUE TITLE
waybar: symbol lookup error: waybar: undifined symbol:

### ISSUE THREAD

**Issue by Pickles888** [Alexays/Waybar#2514]
After executing yay -Syuu, waybar would no longer run and I get the message:
`waybar: symbol lookup error: waybar: undefined symbol: _ZN6spdlog7details7log_msgC1ENS_10source_locEN3fmt3v1017basic_string_viewIcEENS_5level10level_enumES6_` 

EDIT:
Fix: sudo pacman -S spdlog; yay -S spdlog; paru -S spdlog; etc.

**Comment by prayujt** [comment]
I'm having this issue too

**Comment by mike-lloyd03** [comment]
I'm having this issue as well but I installed from official Arch repos.

**Comment by Pickles888** [comment]
thank you! your solution worked!

**Comment by prayujt** [comment]
that didn't work for me - I am using the `spdlog` and `waybar` packages from the Arch repositories as well

**Comment by Pickles888** [comment]
try updating spdlog and if it's on the newest version then try downgrading it maybe?

**Comment by mike-lloyd03** [comment]
It's working for me on latest `waybar` and `spdlog` from Arch repos.

**Comment by prayujt** [comment]
@Pickles888 I'm on `spdlog` 1.12.0, and `waybar` 0.9.22. I can't downgrade `spdlog` since it's saying `'libspdlog.so=1.12-64' required by waybar`

**Comment by mcepl** [comment]
I have something which looks very much similar here on openSUSE/Tumbleweed with waybar 0.9.24, fmt 9.1.0, and spdlog 1.12.0:

```
mitmanek:~ $ waybar
waybar: symbol lookup error: waybar: undefined symbol: _ZN6spdlog7details7log_msgC1ENS_10source_locEN3fmt2v917basic_string_viewIcEENS_5level10level_enumES6_
mitmanek:~ $ 
```

I don’t even know whom to blame and where to file a bug.

**Comment by AnTravel** [comment]
Yeah I'm having the exact same issue as @mcepl on Tumbleweed. Same output on the terminal and everything.

**Comment by Kimitri** [comment]
Yep. Same exact thing here on Tumbleweed. I just upgraded everything and Waybar broke. libspdlog1_12 version 1.12.0-3.2.

**Comment by AnTravel** [comment]
@mcepl @Kimitri I just did a 'sudo zypper dup' and there was an update to waybar. That fixed it for me. How about you guys? Looks like the update just came out.

**Comment by Kimitri** [comment]
@AnTravel Sure enough, it works now. Thanks!
